File: benchmarks/suite5/sum.zig
const bench = @import("bench.zig");
const clib = @cImport({
    @cInclude("stdio.h");
});

pub fn main() void {
    const n: i64 = bench.opaqueI64(1500000);
    var acc: i64 = 0;
    var i: i64 = 0;
    while (i < n) : (i += 1) {
        acc += i * i;
    }
    bench.printI64(acc);
}

File: benchmarks/suite5/sum.c
#include <stdint.h>
#include <stdio.h>
#include "bench.h"

int main(void) {
  const int64_t n = 1500000;
  int64_t acc = 0;
  for (int64_t i = 0; i < n; i++) {
    acc += i * i;
  }
  suite5_print_i64(acc);
  return 0;
}
